dll/win32/comctl32/progress.c

```diff
@@ -652,8 +655,14 @@ static LRESULT WINAPI ProgressWindowProc(HWND hwnd, UINT message,
 	INT oldVal;
         oldVal = infoPtr->CurVal;
         infoPtr->CurVal += infoPtr->Step;
-        if(infoPtr->CurVal > infoPtr->MaxVal)
-	    infoPtr->CurVal = infoPtr->MinVal;
+        if (infoPtr->CurVal > infoPtr->MaxVal)
```

Comment:
This looks like a fix for https://jira.reactos.org/browse/CORE-13873 .

Reply:
It is exactly that fix.